source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,67 @@
+## Trigger automations with your Mac
+
+Home Assistant Companion for macOS adds several new binary sensors for your Mac showing whether it is active and whether a particular microphone or webcam is in use.
+
+The "active" sensor reports whether the Mac is being actively used. In other words that it is not sleeping, not showing a screensaver, not locked and not just sat idle. You can configure the "Time Until Idle" in one-minute steps from a minimum of 1 minute, you'll find this option in the Sensors section of Preferences.
+
+Each camera and microphone has its own `binary_sensor` showing whether it is active or not. These can enable some really useful automations, especially for those home working at the moment. You could automatically turn off the radio when answering a call or close the blinds behind you to improve your video quality. To see just how useful this can be in the real world, check out this video of how our very own Frenck is using these sensors in his streaming set up.
+
+TODO: ADD VIDEO EMBED HERE
+
+One huge advantage of running on a Mac compared to a mobile device is the much larger battery. This means we are not constrained by battery saving measures and can address one of the most common gripes with the iOS app, update intervals. On a Mac entity updates are triggered immediately when something changes. You will see this reported by the `sensor.DEVICE_NAME_last_update_trigger` reporting `Signaled`.
+
+## Notifications
+
+Just like the iOS app, you can send notifications to your Mac with services like `notify.mobile_app_DEVICE_NAME`. One small difference is that [critical notifications](https://companion.home-assistant.io/docs/notifications/critical-notifications) are not yet available for the Mac app. However, all our other notifications features like actionable notifications work on the Mac app. To see what is possible, take a look at [the docs](https://companion.home-assistant.io/).
+
+## Interface
+
+The Mac app is definitely a _Mac_ app. The App Configuration page has been removed from Home Assistant's sidebar and instead the configuration options and preferences are on the menu bar right where you'd expect to find them for any other app and all the standard shortcuts work too (like `⌘,` for Preferences). You can even open multiple Lovelace windows via File>New.
+
+In the menu bar you will also find an option to manually send an update to Home Assistant and a new Actions menu where you can see all your [actions](https://companion.home-assistant.io/docs/core/actions) and fire them.
+
+## Big Sur Features
+
+Home Assistant Companion for macOS already supports widgets in Big Sur. Right now we have an Actions Widget where you can have up to eight actions. You can also create multiple widgets with different sets of actions. If you have an idea for other widgets you'd like to see, pop over to [the community forums and let us know](https://community.home-assistant.io/t/what-kind-of-ios-14-widgets-would-you-like-to-see/211112/14).
```

Comment:
```suggestion
Home Assistant Companion for macOS already supports widgets in Big Sur. Right now, we have an Actions Widget where you can have up to eight actions. You can also create multiple widgets with different sets of actions. If you have an idea for other widgets, you'd like to see, pop over to [the community forums and let us know](https://community.home-assistant.io/t/what-kind-of-ios-14-widgets-would-you-like-to-see/211112/14).
```

Reply:
Agree on first coma, not sure on the second the "you'd like to see" isn't subordinate and it feels weird with the break there (IMO)

Reply:
🤷‍♂️ Fine either way. The last one is more personal I guess.

Reply:
Done. Agree, really appreciate the level of attention readability to a global audience has to come first! 🙂